blog/2024-01-24-news-jan-2024/index.qmd

```diff
@@ -0,0 +1,44 @@
+
+We also have a [February Community Call](TODO LINK): **GitHub for NASA Openscapes community project management & calendaring - a work in progress**. We invite you to come see how the NASA Openscapes Mentors are co-organizing an open calendar and shared tasks — bring your questions and experience!
+
+We'll be sharing more events and goals for 2024 in the coming months.
+
+
+## Some 2023 Highlights
+
+We are continually iterating and documenting what works as we learn from experiences collaborating and teaching. This year we've focused on communicating this in a few talks, particularly [the Flywheel as an open source tool](https://openscapes.org/blog/2024-01-18-agu-talk-2023/) as we described in a talk at the AGU meeting and in [documenting things, particularly onboarding](https://openscapes.org/blog/2023-09-27-documenting-things-posit-conf/) at the Posit conference in September.
+
+[Openscapes Mentors](https://openscapes.org/team) were an important part of the Year of Open Science. Together, across NASA Earthdata, NOAA Fisheries, California Water Boards, Pathways to Open Science, EPA, and Fred Hutch, we learned [coaching skills that have made us better open data science mentors](https://openscapes.org/blog/2023-05-17-mentor-coach/), and co-wrote a preprint [ Shifting institutional culture to develop climate solutions with Open Science](https://eartharxiv.org/repository/view/5948/) that is in review.
+
+[NASA Openscapes](https://nasa-openscapes.github.io) progress focused on learning and codeveloping with the broader open science community via hackdays focused on R, MATLAB, Cookbook, and 2i2c access policies. Mentors across 11 NASA data centers continue improving how we teach and support researchers in the Cloud via workshops, hackathons, and Champions Cohorts. California Water Boards supported [their second annual Champions cohort](https://openscapes.org/blog/2023-12-13-waterboards/) to support their colleagues to meet open, data, and equity goals. NOAA Fisheries shared their Open Data and Open Science vision at the summer ESIP meeting! 
```

Comment:
```suggestion
[NASA Openscapes](https://nasa-openscapes.github.io) progress focused on learning and codeveloping with the broader open science community via hackdays focused on R, MATLAB, the [Earthdata Cloud Cookbook](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/), and 2i2c Openscapes JupyterHub access policies. Mentors across 11 NASA data centers continue improving how we teach and support researchers in the Cloud via workshops, hackathons, and [Champions Cohorts](https://openscapes.org/blog/2023-08-01-nasa-champions/). It was truly energizing to come together in person with so many NASA Openscapes Mentors and collaborators at the summer ESIP and winter [AGU meetings](https://openscapes.org/blog/2023-12-10-agu-fall-meeting/)!

California Water Boards ran [their second annual Champions Cohort](https://openscapes.org/blog/2023-12-13-waterboards/) to support their colleagues to meet open, data, and equity goals. NOAA Fisheries shared their Open Data and Open Science vision at the summer ESIP meeting! 
```

Reply:
add Carpentries Instructor training. How many certified?

and [Engaging with a Cross-NASA Subcommunity
Recap of a Carpentries Community Session with NASA and Openscapes](https://openscapes.org/blog/2023-12-07-engaging-with-a-cross-nasa-subcommunity/)

Reply:
need cohesion for paragraph starting "California Water Boards...". Maybe it belongs above with "[Openscapes Mentors](https://openscapes.org/team) were an important part of the Year of Open Science. Together, across NASA Earthdata, NOAA Fisheries, California Water Boards, ..." that refers to cohorts and mentors activities.